browser-test/package.json

```diff
@@ -16,5 +16,8 @@
     "ts-node": "10.8.1",
     "ts-node-dev": "2.0.0",
     "typescript": "4.7.4"
+  },
+  "dependencies": {
```

Comment:
Not blocking: Two issues around consistency with other dependencies.

1. Any reason we shouldn't put this in devDependencies?
2. Should we use a specific version number rather than using "compatible-with" semantics?

Reply:
done